### DISCUSSION TITLE
color报错

### DISCUSSION THREAD

**Discussion by albert99zhy** [Linaom1214/TensorRT-For-YOLO-Series#110] [Q&A]
位置：color=（_COLORS[cls_id]*255）.astype（np.uint8）.tolist（）
报错：Indexerror ：index 15675 is out of bounds for axis 0 with size 80
请问这个报错改如何更改？

**Comment by Linaom1214** [comment]
应该是模型输出错了， 使用end2end推理的时候 要输入-end2end ---- 回复的原邮件 ***@***.***>发送日期2023年10月22日 19:08 ***@***.***> ***@***.***>主题[Linaom1214/TensorRT-For-YOLO-Series] color报错 (Discussion #110)
位置：color=（_COLORS[cls_id]*255）.astype（np.uint8）.tolist（）
报错：Indexerror ：index 15675 is out of bounds for axis 0 with size 80
请问这个报错改如何更改？

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/Linaom1214/TensorRT-For-YOLO-Series/discussions/110",
"url": "https://github.com/Linaom1214/TensorRT-For-YOLO-Series/discussions/110",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
